src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy

```diff
@@ -276,50 +307,78 @@ class PvModelTest extends Specification {
 
     "== Calculate the angle of incidence thetaG =="
-    when:
+    given:
     // Declination Angle delta of the sun at solar noon
-    Angle delta = Sq.create(Math.toRadians(-14), Radians$.MODULE$)
+    Angle deltaRad = Sq.create(Math.toRadians(deltaIn), Radians$.MODULE$)
     //Latitude in Radian
-    Angle latitudeInRad = Sq.create(Math.toRadians(43d), Radians$.MODULE$)
+    Angle latitudeInRad = Sq.create(Math.toRadians(latitudeInDeg), Radians$.MODULE$)
     //Hour Angle
-    Angle omega = Sq.create(Math.toRadians(-22.5), Radians$.MODULE$)
+    Angle omegaRad = Sq.create(Math.toRadians(omegaDeg), Radians$.MODULE$)
     //Inclination Angle of the surface
-    Angle gammaE = Sq.create(Math.toRadians(45), Radians$.MODULE$)
+    Angle gammaERad = Sq.create(Math.toRadians(gammaEDeg), Radians$.MODULE$)
     //Sun's azimuth
-    Angle alphaE = Sq.create(Math.toRadians(15), Radians$.MODULE$)
+    Angle alphaERad = Sq.create(Math.toRadians(alphaEDeg), Radians$.MODULE$)
+
+    when:
+    Angle thetaG = pvModel.calcAngleOfIncidenceThetaG(deltaRad, latitudeInRad, gammaERad, alphaERad, omegaRad)
 
     then:
+    "- should calculate the angle of incidence thetaG "
+    thetaG.toDegrees() =~ Sq.create(thetaGOut, Degrees$.MODULE$).toDegrees()
 
-    pvModel.calcAngleOfIncidenceThetaG(delta, latitudeInRad, gammaE, alphaE, omega).toDegrees() =~ 35.176193345578604
+    where: "the following parameters are given"
+    latitudeInDeg | deltaIn         | omegaDeg       | gammaEDeg  | alphaEDeg  || thetaGOut
+    43d           | -14d            | -22.5d         | 45d        | 15d        || 35.176193345578606393727080835951995075234213360724d  // Duffie
+    51.516667d    | +18.4557514d    | -15.00225713d  | 30d        | +0d        || 14.420271449960715d                     // Iqbal
+    51.516667d    | +18.4557514d    | -15.00225713d  | 90d        | +0d        || 58.65287310017624d                     // Iqbal
+    35.0d         | +23.2320597d    | +30.00053311d  | 45d        | 10d        || 39.62841449023577d                      // Kalogirou - Solar Energy Engineering Example 2.7  ISBN 978-0-12-374501-9; DOI https://doi.org/10.1016/B978-0-12-374501-9.X0001-5
+    35.0d         | +23.2320597d    | +30.00053311d  | 45d        | 90d        || 18.946300807438607d                     // Kalogirou - Solar Energy Engineering Example 2.7 changed to 90° panel azimuth to WEST
+    35.0d         | +23.2320597d    | +74.648850625d | 45d        | 90d        || 21.95480347380729d                     // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to WEST at 17:00
+    35.0d         | +23.2320597d    | +74.648850625d | 45d        | -90d       || 109.00780288303966d                     // Kalogirou - Solar Energy Engineering Example 2.7  90° panel azimuth to EAST at 17:00
+    27.96d        | -17.51d         | -11.1d         | 30d        | +10d       || 22.384603601536398d                     // Goswami Priciples of Solar Engineering Example 2.7a
```

Comment:
In Goswami that's called panel azimuth angle -> Maybe it's something different?

Reply:
Which exactly do you mean? Panel azimuth angle @ Goswami is alpha_w which represents the rotation angle between the east-west axis and the panel (Figure 2.21). That corresponds to our defintion. A panel with alpha_w = +10° should be turned slightly to the east from the south.

Reply:
Oh, I think I have mixed up the names of the angles. Yours should be right.